.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
```

Comment:
if this is `*`, then the editorconfig check should also use `*` - if it's just js/jsx, then i'd like both places to match.